docs/pages/release_notes.md

```diff
@@ -39,6 +39,9 @@ Both are bugfixing releases.
 
```

Comment:
In the past, we also shortly mentioned the new CLI flags introduced in the release notes.
I don't think it's under "new and noteworthy", but maybe just a bullet point under API Changes? (Since CLI is also an API).

Reply:
That makes sense, thanks